infra.bs

```diff
@@ -486,6 +486,24 @@ implementations of just <a>JavaScript strings</a> for performance and memory rea
 
 <hr>
 
+<p>To <dfn export for=string lt=concatenate|concatenation>concatenate</dfn> a <a for=/>list</a> of
+<a for=/>strings</a> <var>list</var>, with an optional separator string <var>separator</var>, run
```

Comment:
I wonder if "using" would be better than "with". "The concatenation of X with Y" almost sounds like the result should be "XY"

Reply:
Good call.